docs/man/git-lfs-config.1.ronn

```diff
@@ -0,0 +1,92 @@
+git-lfs-config(1) -- Configuration options for git-lfs
```

Comment:
I'm certainly no `man` expert, but I think 5 is a more appropriate man section: "File formats and conventions" (https://en.wikipedia.org/wiki/Man_page). Git tracks the config values in [git-config(1)](https://www.kernel.org/pub/software/scm/git/docs/git-config.html), but that goes with the `git config` command. Hooks with no related command are documented in [githooks(5)](https://www.kernel.org/pub/software/scm/git/docs/githooks.html).